src/classes/classes-protected.ts

```diff
@@ -0,0 +1,41 @@
+//protected classes
+
+class Employee {
+  constructor(protected identifier: string) {}
```

Comment:
this is syntactic sugar and must be explained before used. Create an extra lesson for this called `classes-fields.ts` and explain there that: 

<img width="622" alt="image" src="https://github.com/geogeorgiou/ts-course/assets/53356354/483eea49-103d-4601-abe4-3049276456cb">

equals to this (delete property definition, delete implementation at the constructor == less code!) 

<img width="770" alt="image" src="https://github.com/geogeorgiou/ts-course/assets/53356354/692c10eb-d82b-4156-92a5-6cc4a0426097">


[Reference video](https://www.udemy.com/course/typescript-the-complete-developers-guide/learn/lecture/15066544#overview)